⬆️ 🎨 Allow the use of gtdb taxonomy in Autometa (#284)
KwanLab/Autometa: docs/source/databases.rst

    autometa-config \
        --section databases --option gtdb \
        --value <path/to/your/gtdb/database/directory>

    # To use a specific GTDB release
    autometa-config \
        --section gtdb --option release \
        --value latest
        # Or --value r207 or --value r202, etc.

    # Download and format the configured GTDB databases release
    autometa-update-databases --update-gtdb

.. note::

    You can check the default config paths using ``autometa-config --print``.

    See ``autometa-update-databases -h`` and ``autometa-config -h`` for full list of options.

The previous command will download the following GTDB databases and format the `gtdb_proteins_aa_reps.tar.gz` to be used by Autometa:

- Amino acid sequences of representative genome
    - `gtdb_proteins_aa_reps.tar.gz <https://data.gtdb.ecogenomic.org/releases/latest/genomic_files_reps/gtdb_proteins_aa_reps.tar.gz>`_
- gtdb-taxdump.tar.gz from `shenwei356/gtdb-taxdump <https://github.com/shenwei356/gtdb-taxdump/releases>`_
    - `gtdb-taxdump.tar.gz <https://github.com/shenwei356/gtdb-taxdump/releases/latest/download/gtdb-taxdump.tar.gz>`_

> Once unzipped `gtdb-taxdump.tar.gz` will have the taxdump files of all the respective GTDB releases. Make sure that the release you use is in line with the `gtdb_proteins_aa_reps.tar.gz` release version. It's better to always use the latest version. 

Reviewer: This section is not clear and you should break this in to multiple lines.

Where does `gtdb.faa` come in? When is `gtdb.dmnd` generated, if at all?

> All the taxonomy files for a specific taxonomy database should be in a single directory. You can now copy the taxdump files of the desired release version in the sample directory as `gtdb.dmnd`

Reviewer: I'm not sure I follow this statement.

What does this mean?

>You can now copy the taxdump files of the desired release version in the sample directory as `gtdb.dmnd`

Alternatively if you have manually downloaded `gtdb_proteins_aa_reps.tar.gz` and `gtdb-taxdump.tar.gz` you can run the following command to format the `gtdb_proteins_aa_reps.tar.gz` file and make it ready for Autometa.

.. code-block:: bash

>     python -m autometa.taxonomy.gtdb --reps-faa <path/to/gtdb_proteins_aa_reps.tar.gz> --dbdir <path/to/output_directory> --cpus 20

Reviewer: Is there an entrypoint in `setup.py` to run this as a module? The `python -m autometa.taxonomy.gtdb` command may only work if the user has installed from source.

.. note::

>     Again Make sure that the formatted `gtdb_proteins_aa_reps.tar.gz` databse and gtdb taxdump files are in the same directory. 

Reviewer: Is the 'formatted `gtdb_proteins_aa_reps.tar.gz` database `gtdb.faa`? 

```suggestion
    The formatted `gtdb_proteins_aa_reps.tar.gz` database and GTDB taxdump files should be in the same directory. 
```
